api/src/test/java/com/epam/pipeline/test/acl/AbstractAclTest.java

```diff
@@ -107,8 +120,35 @@ protected void initAclEntity(AbstractSecuredEntity entity, List<AbstractGrantPer
         doReturn(acl).when(aclService).getOrCreateObjectIdentity(eq(entity));
         doReturn(acl).when(aclService).createAcl(eq(entity));
         doReturn(acl).when(aclService).updateAcl(acl);
+        doReturn(Collections.singletonMap(objectIdentity, acl)).when(aclService).getObjectIdentities(anySet());
+    }
+
+    protected void initAclEntity(List<Pair<AbstractSecuredEntity, List<AbstractGrantPermission>>> pairs) {
+        final Map<ObjectIdentity, Acl> acls = new HashMap<>();
+        pairs.forEach(p -> {
+            final AbstractSecuredEntity entity = p.getLeft();
+            final List<AbstractGrantPermission> permissions = p.getRight();
+            initAclEntity(entity, permissions);
+            ObjectIdentityImpl objectIdentity = new ObjectIdentityImpl(entity);
+            AclImpl acl = prepareAcl(objectIdentity, entity, permissions);
+            acls.put(objectIdentity, acl);
+        });
+        doReturn(acls).when(aclService).getObjectIdentities(anySet());
+    }
```

Comment:
I would suggest getting rid of an additional `prepareAcl` method but just return `AclImpl` from an existing `initAclEntity` method. Once such refactoring is applied then we can simplify new method like this:

```suggestion
    protected void initAclEntity(List<Pair<AbstractSecuredEntity, List<AbstractGrantPermission>>> pairs) {
        final Map<ObjectIdentity, Acl> acls = pairs.stream()
            .map(p -> initAclEntity(p.getLeft(), p.getRight()))
            .collect(Collectors.toMap(AclImpl::getObjectIdentity, Function.identity());
        doReturn(acls).when(aclService).getObjectIdentities(anySet());
    }
```